images/Dockerfile.ubuntu

```diff
@@ -13,11 +13,12 @@ ARG FLAVOR_RELEASE
 #   - rpi4
 #   - rpi3
 ARG MODEL=generic
-# HWE is used to determine if the HWE kernel should be installed
-# the default value is empty, which means the HWE kernel WILL be installed
-# if you want to disable the HWE kernel, set HWE to "-non-hwe"
-ARG HWE=""
```

Comment:
I removed all concept of HWE since now the user can pass a non-hwe image instead 🎉

Reply:
I see you also added an example for that image. Maybe we should document how to to use it (kairos-docs PR?)

Reply:
yup indeed, will create an issue to not forget

Reply:
https://github.com/kairos-io/kairos/issues/2036